File: week-2/hollow-square.c
#include <stdio.h>

void secret_shape(int n)
{
   for(int i = 1; i <= (2*n-1); i++)
   {
     for(int j = 1; j <= (2*n-1); j++)
     {
       if(j == 1 || j == (2*n-1) || i==1 || i == (2*n-1)) printf("*");
       else printf(" ");
     }
     printf("\n");
   }
   printf("\n");
}

int main()
{
   int start = 1;
   int end = 5;
   for(int i = 1; i <= 5; i++)
   {
      secret_shape(i);
   }
   return 0;
}

File: week-2/hollow_square.py
def secret_shape(n):
    for i in range(1,2*n):
        for j in range(1,2*n):
            if (i == 1 or j == 1 or i == (2*n-1) or j == (2*n-1)):
                print("*",end="")
            else:
                print(" ",end="")
        print()
# code for testing
for i in range(1,6):
    secret_shape(i)
    print()
